web/docs/language/features.md

```diff
@@ -443,6 +443,183 @@ import { isPrismaError, prismaErrorToHttpError } from '@wasp/utils.js'
   }
 ```
 
+## Jobs
+
+If you have tasks that you do not want to handle as part of the normal request-response cycle, Wasp allows you to make that function a `job` and it will gain some "superpowers." Jobs will persist between server restarts, can be retried if they fail, and they can even be delayed until the future (or have a recurring schedule)! Some examples where you may want to use a `job` include sending an email, making an HTTP request to some external API, or doing some nightly calculations.
+
+### Job Executors
+
+Job executors handle the scheduling, monitoring, and execution of our jobs.
+
+Wasp allows you to choose which job executor will be used to execute a specific job that you define, which affects some of the finer details of how jobs will behave and how they can be further configured. Each job executor has its pros and cons, which we will explain in more detail below, so you can pick the one that best suits your needs.
+
+Currently, Wasp supports only one type of job executor, which is `PgBoss`, but in the future, it will likely support more.
+
+#### pg-boss
+
+We have selected [pg-boss](https://github.com/timgit/pg-boss/) as our first job executor to handle the low-volume, basic job queue workloads many web applications have. By using PostgreSQL (and [SKIP LOCKED](https://www.2ndquadrant.com/en/blog/what-is-select-skip-locked-for-in-postgresql-9-5/)) as its storage and synchronization mechanism, it allows us to provide many job queue pros without any additional infrastructure or complex management.
```

Comment:
Should we mention here, before details, in very simple vocabulary, what are the limitations of pg-boss executor as is currently implemented in Wasp?
That would mean mentioning that they should be careful with CPU tasks as they are not executed separately from the rest of the server logic but they share CPU. And maybe smth else, are there any other things to keep in mind?

Reply:
Yeah, I can sneak a few of the main things as headlines into a line before 👍🏻